modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/get_data_attributes.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+export default function getDataAttributes(data) {
```

Comment:
We could simplify this further by having something like `getDropdownItems`, that would do mapping as well. That code is shared by all three cards as well as `DropdownMenu`. Name of the file would have to be different. For a new util we should also have jest test coverage. I would honestly leave it all for a separate task.

Reply:
:+1: that all sounds like good ideas to me.